targets/targets.json

```diff
@@ -1988,6 +1988,36 @@
         "release_versions": ["2", "5"],
         "device_name": "STM32F411RE"
     },
+    "MTS_DRAGONFLY_L471QG": {
+        "inherits": ["FAMILY_STM32"],
+        "supported_form_factors": ["ARDUINO"],
+        "core": "Cortex-M4F",
+        "extra_labels_add": ["STM32L4", "STM32L471QG", "STM32L471xG", "STM32L471xx"],
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSI | USE_PLL_MSI",
+                "value": "USE_PLL_MSI",
+                "macro_name": "CLOCK_SOURCE"
+            },
+            "modem_is_on_board": {
+                "help": "Value: Tells the build system that the modem is on-board as oppose to a plug-in shield/module.",
+                "value": 1,
+                "macro_name": "MODEM_ON_BOARD"
+            },
+            "modem_data_connection_type": {
+                "help": "Value: Defines how an on-board modem is wired up to the MCU, e.g., data connection can be a UART or USB and so forth.",
+                "value": 1,
+                "macro_name": "MODEM_ON_BOARD_UART"
+            }
+        },
+        "detect_code": ["0312"],
+        "macros_add": [ "RTC_LSE=1", "CELLULAR_DEVICE=UBLOX_PPP"],
```

Comment:
`CELLULAR_DEVICE` need not be defined here, that's defined in `CellularTargets.h`

Reply:
Yes, remove from here.

Reply:
RTC_LSE is no more used in TARGET_STM

Reply:
I will be pushing a commit soon. I removed RTC_LSE=1. In CellularTargets.h, I see that CELLULAR_DEVICE is being defined as UBLOX_PPP. However, if I remove "CELLULAR_DEVICE=UBLOX_PPP" from targets.json, it fails to compile at least some tests.